.github/workflows/run_test_suite.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
           ipfs_version: v0.17.0
           run_daemon: true
       - name: 'Run Rust native target tests'
-        run: cargo test --features test-kubo,helpers,performance
+        run: cargo test --features test-kubo,helpers
```

Comment:
It might be interesting to run the benchmark in its own job, just to make sure we're keeping it in a working state.

Reply:
that's not necessary, it is still compiled and linted in CI